### PR TITLE
README.md: Move example project to _projects/example.md

### PR DESCRIPTION
**Pull request by himanshu1603**
This moves example project written on README.md to _projects under the name example.md and excluded from appearing in the project list.

Closes https://github.com/coala/projects/issues/484

**For short term contributors:** we understand that getting your commits well
defined like we require is a hard task and takes some learning. If you
look to help without wanting to contribute long term there's no need
for you to learn this. Just drop us a message and we'll take care of brushing
up your stuff for merge!

### Checklist

- [x] I read the [commit guidelines](http://coala.io/commit) and I've followed
      them.
- [x] I ran coala over my code locally. (*All commits have to pass
      individually.* It is not sufficient to have "fixup commits" on your PR,
      our bot will still report the issues for the previous commit.) You will
      likely receive a lot of bot comments and build failures if coala does not
      pass on every single commit!

After you submit your pull request, **DO NOT click the 'Update Branch' button.**
When asked for a rebase, consult [coala.io/rebase](https://coala.io/rebase)
instead.

Please consider helping us by reviewing other peoples pull requests as well:

- pick up any PR at <https://coala.io/review>
- review it (check <https://coala.io/reviewing> for more info)
- if you are sure that it needs work, use `cobot mark wip <URL>` to get it out
  of the review queue.

The more you review, the more your score will grow at coala.io and we will
review your PRs faster!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -45,59 +45,11 @@ Then you can simply go to either of the following addresses in your browser to a
 
 If you face problems while installing Jekyll or using its gem bundler you may go through its [troubleshooting docs](https://jekyllrb.com/docs/troubleshooting/)
```
Comment (virresh) [inline README.md]: Should there be a reference to the example markdown file here ?
(Something like - "A sample project definition is availaible here (with a hyperlink)" ,Since this example file won't be visible on the projects page and is also not referred anywhere else, it might get hard for people to find the example.md)

File: _projects/example.md [hunk 2]
```diff
@@ -0,0 +1,45 @@
+## Sample project
+
+```
```
Comment (virresh) [inline _projects/example.md]: Consider removing from line 1 to line 3 
and line 45

Refer to any of our project file : 
https://raw.githubusercontent.com/coala/projects/master/_projects/aspects.md
and how it's rendered by github markdown
https://github.com/coala/projects/blob/master/_projects/aspects.md

**Comment by himanshu1603** [comment]
Please review the changes

File: README.md [hunk 3]
```diff
@@ -27,10 +27,11 @@ Why?
 - Stop wasting time maintaining a mentors list. This can be generated from the
   projects.
 
+
```
Comment (virresh) [inline README.md]: Is the addition of this line intentional ?

**Comment by himanshu1603** [comment]
Please check the changes.

**Comment by himanshu1603** [comment]
Thanks!

File: README.md [hunk 4]
```diff
@@ -24,13 +24,12 @@ Why?
   the full information only when needed.
 - A proper review process can be used for triaging and iterating on project
   ideas.
-- Stop wasting time maintaining a mentors list. This can be generated from the
-  projects.
+- Stop wasting time maintaining a mentors list. This can be generated from the projects.
```
Comment (jayvdb) [inline README.md]: revert unrelated change.

File: README.md [hunk 5]
```diff
@@ -24,13 +24,12 @@ Why?
...
 
 ## Usage
 
 To clone the repository and run this website on your local machine, [install Jekyll](https://jekyllrb.com/docs/installation/) for your OS and type the following commands:
-				
+
```
Comment (jayvdb) [inline README.md]: unrelated change.

File: README.md [hunk 6]
```diff
@@ -47,57 +46,14 @@ If you face problems while installing Jekyll or using its gem bundler you may go
...
-* No exceptions to that. Changing the goals is possible together with mentors.
-* Yes, that includes tests and documentation.
-
-##### GSOC 2017 FINAL
-
-* Everything has to be reviewed and merged.
-* Including tests and docs, again.
-
-```
+Mentors and admins can define projects in markdown using the following structure and save it in the _projects folder with a relevant filename.
```
Comment (jayvdb) [inline README.md]: break lines at 80 cpl

` _projects .. ` creates italics.  needs to be in backticks.

File: README.md [hunk 7]
```diff
@@ -47,57 +46,14 @@ If you face problems while installing Jekyll or using its gem bundler you may go
...
+A sample project definition is available [here](https://github.com/coala/projects/blob/master/_projects/example.md)
 
 ## Defining FAQs
 
 Users can also add FAQs by simply creating a markdown file in _faq folder.
 
-Format for faq markdown file is as follows: 
+Format for faq markdown file is as follows:
```
Comment (jayvdb) [inline README.md]: unrelated change

**Comment by himanshu1603** [comment]
please have a look at these changes

File: README.md (hunk 5 above)
Comment (jayvdb) [reply README.md]: still unrelated change here; pls revert

File: README.md [hunk 8]
```diff
@@ -30,14 +30,13 @@ Why?
...
 To clone the repository and run this website on your local machine, [install Jekyll](https://jekyllrb.com/docs/installation/) for your OS and type the following commands:
-				
+
     $ sudo gem install jekyll bundler
     $ git clone https://github.com/coala/projects.git
     $ cd projects
     $ bundle install
     $ bundle exec jekyll serve
 
-
```
Comment (jayvdb) [inline README.md]: still unrelated change here; pls revert

File: README.md [hunk 9]
```diff
@@ -47,57 +46,14 @@ If you face problems while installing Jekyll or using its gem bundler you may go
...
+Mentors and admins can define projects in markdown using the following structure and save it in the ``_projects`` folder with a relevant filename.
+A sample project definition is available [here](https://github.com/coala/projects/blob/master/_projects/example.md)
 
 ## Defining FAQs
 
-Users can also add FAQs by simply creating a markdown file in _faq folder.
+Users can also add FAQs by simply creating a markdown file in ``_faq`` folder.
 
-Format for faq markdown file is as follows: 
+Format for faq markdown file is as follows:
```
Comment (jayvdb) [inline README.md]: still unrelated change here; pls revert

File: README.md [hunk 10]
```diff
@@ -47,57 +46,14 @@ If you face problems while installing Jekyll or using its gem bundler you may go
...
-* Including tests and docs, again.
-
-```
+Mentors and admins can define projects in markdown using the following structure and save it in the ``_projects`` folder with a relevant filename.
+A sample project definition is available [here](https://github.com/coala/projects/blob/master/_projects/example.md)
 
 ## Defining FAQs
 
-Users can also add FAQs by simply creating a markdown file in _faq folder.
+Users can also add FAQs by simply creating a markdown file in ``_faq`` folder.
```
Comment (jayvdb) [inline README.md]: still unrelated change here; pls revert

**Comment by himanshu1603** [comment]
does this looks good now

**Comment by himanshu1603** [comment]
Is it correct?

**Comment by jayvdb** [comment]
No.  this is getting worse.
See https://github.com/coala/projects/pull/543/commits .  You now have 6 commits.
We want one commit.  And it should only have the changes which are needed.

Read https://github.com/coala/projects/pull/543/files carefully.

Read http://api.coala.io/en/latest/Developers/Git_Basics.html#squashing-your-commits

**Comment by jayvdb** [comment]
I've pushed the fix as this is wasting our time, and we have other changes which are dependent on this one.

**Comment by jayvdb** [comment]
ack https://github.com/coala/projects/pull/543/commits/729dde2817760314b7af11644c3b214d71cb3dbc

**Comment by jayvdb** [comment]
@gitmate-bot ff
